clr-k8s-examples/1-core-metrics/metrics-server-deployment.yaml

```diff
@@ -22,10 +23,19 @@ spec:
         k8s-app: metrics-server
     spec:
       serviceAccountName: metrics-server
+      volumes:
+      # mount in tmp so we can safely use from-scratch images and/or read-only containers
+      - name: tmp-dir
+        emptyDir: {}
       containers:
       - name: metrics-server
-        image: gcr.io/google_containers/metrics-server-amd64:v0.2.0
+        image: k8s.gcr.io/metrics-server-amd64:v0.3.1
         imagePullPolicy: Always
-        command:
-        - /metrics-server
-        - --source=kubernetes.summary_api:https://kubernetes.default.svc?kubeletHttps=true&kubeletPort=10250&useServiceAccount=true&insecure=true
+        args:
+        - --logtostderr
+        - --kubelet-insecure-tls
```

Comment:
@krsna1729 is this a limitation. This was due to a bug right.

Reply:
This was because of self-signed certs IIRC